pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```
@@ -13,6 +13,8 @@ in
 
   agda2-mode = callPackage ./manual-packages/agda2-mode { };
 
+  beancount-mode = callPackage ./manual-packages/beancount-mode {};
```

```suggestion
  beancount = callPackage ./manual-packages/beancount {};
```

Since this package has `(provide 'beancount)`, by convention, it should be named `beancount`.